### PR TITLE
Sxmo 1.7

### PR DESCRIPTION
**Pull request by ArenM**
I've been using this for a while and it seems pretty stable.

The image build script might need a corresponding change because:
 - sxmo-ui-sway-meta now depends on sway instead of sxmo-sway. The patches in sxmo-sway haven't landed, but it is very useful to be able to install other sway versions for testing
 - vis has been changed to an optional dependency, it's still required for the default editor

### CODE REVIEW

**Comment by hamblingreen1** [comment]
update and install of wob  and sxmo-utils went well. wob doesn't show bar when changing volume or brightness.

File: PKGBUILDS/sxmo/sxmo-utils/PKGBUILD [hunk 1]
```diff
@@ -20,16 +20,22 @@ backup=('etc/doas.conf')
...
-  # Use dunst instead because of issues between mako and sxmo-sway on arch
-  sed -i 's@mako@dunst@g' configs/appcfg/sway_template
-
   # Use systemd to start & stop user services
+  # TODO: this should be upstreamed, the new sxmo_daemons (likely in 1.8) script
+  # should make that easier
   sed -e 's@setsid -f mmsdtng@systemctl --user start mmsd-tng.service@g' \
       -e 's@pkill mmsdtng@systemctl --user stop mmsd-tng.service@g' \
       -i scripts/core/sxmo_mmsdconfig.sh
+  sed -i '/mmsdtng/d' configs/default_hooks/start
```
Comment (oo-san) [inline PKGBUILDS/sxmo/sxmo-utils/PKGBUILD:29]: This creates an empty if block in the start script. Consider replacing with 
`sed -i 's/mmsdtng.*/:/' configs/default_hooks/start`

File: PKGBUILDS/sxmo/sxmo-utils/PKGBUILD [hunk 2]
```diff
@@ -20,16 +20,22 @@ backup=('etc/doas.conf')
...
+  sed -i '/pkill mmsdtng/d' configs/default_hooks/stop
+  sed -i '/checkmmsd/a return' scripts/modem/sxmo_mms.sh
 
-  # the unclutter executable has a different name on arch than on alpine
-  sed -i 's@unclutter-xfixes@unclutter@g' configs/appcfg/xinit_template
+  sed -e 's@setsid -f vvmd@systemctl --user start vvmd.service@g' \
+      -e 's@pkill vvmd@systemctl --user stop vvmd.service@g' \
+      -i scripts/core/sxmo_mmsdconfig.sh
+  sed -i '/vvmd/d' configs/default_hooks/start
```
Comment (oo-san) [inline PKGBUILDS/sxmo/sxmo-utils/PKGBUILD:36]: This creates an empty if block in the start script. Consider replacing with 
`sed -i 's/vvmd.*/:/' configs/default_hooks/start`

**Comment by oo-san** [comment]
Also works well for the things that I tested related to the default ui. I have not tested dwm or anything related to dialing yet.

For me, using the default start hook through `sxmo_migrate` produces a syntax error due to the empty `if` blocks for `mms` and `vvm`. This prevents `wob` from executing. Commenting those out or putting a colon in the if block fixes it for me.

**Comment by hamblingreen1** [comment]
> Also works well for the things that I tested related to the default ui. I have not tested dwm or anything related to dialing yet.
Can confirm that ui changes work just fine on dwm, tested that the other day.
> For me, using the default start hook through sxmo_migrate produces a syntax error due to the empty if blocks for mms and vvm. This prevents wob from executing. Commenting those out or putting a colon in the if block fixes it for me.
Also confirming, this fixes wob. earlier i got it to work by manually invoking it with the cli, but this fix makes wob behave as intended by sxmo

File: PKGBUILDS/sxmo/sxmo-utils/PKGBUILD (hunk 1 above)
Comment (ArenM) [reply PKGBUILDS/sxmo/sxmo-utils/PKGBUILD]: Good catch, applied

File: PKGBUILDS/sxmo/sxmo-utils/PKGBUILD (hunk 2 above)
Comment (ArenM) [reply PKGBUILDS/sxmo/sxmo-utils/PKGBUILD]: Applied

**Comment by Danct12** [comment]
LGTM, build successfully on the server as well.

**Pull request merged**